crates/re_string_interner/src/lib.rs

```diff
@@ -8,6 +8,8 @@
 //!
 //! The interned strings are never freed, so don't intern too many things.
 
+pub use nohash_hasher;
```

Comment:
Let's be consistent in the way we reexport things.
```suggestion
pub mod external {
    pub use nohash_hasher;
}